docs/operation/operation.md

```diff
@@ -505,6 +620,83 @@ The following timer metrics are exposed per used bundle-name:
 - `skipper.opaAuthorizeRequest.custom.eval_time.<bundle-name>`
 - `skipper.opaServeResponse.custom.eval_time.<bundle-name>`
 
+### RouteSRV metrics
+
+RouteSRV metrics expose the following metrics in Prometheus format:
+
+```
+% curl http://127.0.0.1:9911/metrics
+# 8< Go metrics >8
+
+# HELP routesrv_backend_combined_duration_seconds Duration in seconds of a proxy backend combined.
+# TYPE routesrv_backend_combined_duration_seconds histogram
+routesrv_backend_combined_duration_seconds_bucket{version="v0.18.38-8",le="0.005"} 5
+routesrv_backend_combined_duration_seconds_bucket{version="v0.18.38",le="0.01"} 5
```

Comment:
The version label would not be there if you drop the linking change.
We would need to tell how to create the version label @AlexanderYastrebov by prometheus relabel config